packages/cardano-services/src/Http/util.ts

```diff
@@ -0,0 +1,19 @@
+import { ProviderError, ProviderFailure } from '@cardano-sdk/core';
+
+export const ORIGIN = 'Origin';
+export const CONTENT_TYPE = 'Content-Type';
+export const APPLICATION_JSON = 'application/json';
+
+export const getListen = (url: URL) => ({ host: url.hostname, port: url ? Number.parseInt(url.port) : undefined });
```

Comment:
I haven't changed this function, just moved it into another `util` file.

Reply:
From memory, it's not strictly required since we're just shadowing the express default, but yes it makes sense in case the underlying default changes.